README.md

```diff
@@ -5,9 +5,11 @@ This project is where the [Open Container Initiative](http://www.opencontainers.
 Table of Contents
 
 - [Filesystem Bundle](bundle.md)
-- [Container Configuration](config.md)
+- [Host-independent Container Configuration](config.md)
   - [Linux Specific Configuration](config-linux.md)
-- [Runtime and Lifecycle](runtime.md)
+- [Host-specific Container Configuration](runtime.md)
+  - [Linux Specific Configuration](runtime-linux.md)
```

Comment:
On Sat, Aug 29, 2015 at 01:11:34AM -0700, Lai Jiangshan wrote:
> >    - [Linux Specific Configuration](config-linux.md)
> > -- [Runtime and Lifecycle](runtime.md)
> > +- [Host-specific Container Configuration](runtime.md)
> > +  - [Linux Specific Configuration](runtime-linux.md)
> 
> you have two `Linux Specific Configuration` one for the
> `config-linux.md` and the other for `runtime-linux.md`

They share the same title because they are namespaced by the
higher-level entry.  To be completely specific, it would be:

  - [Host-independent Container Configuration](config.md)
    - [Linux-specific, Host-independent Configuration](config-linux.md)
  - [Host-specific Container Configuration](runtime.md)
    - [Linux-specific, Host-specific Configuration](runtime-linux.md)

But that seemed too wordy.

Although I suppose I should at least convert to “Linux-specific” for
consistent hyphenation.